presto-main/src/main/java/com/facebook/presto/cost/PlanNodeCostEstimate.java

```diff
@@ -124,7 +124,7 @@ public static PlanNodeCostEstimate cpuCost(double cpuCost)
 
     public static PlanNodeCostEstimate memoryCost(double memoryCost)
     {
-        return builder().setCpuCost(memoryCost).build();
+        return builder().setMemoryCost(memoryCost).build();
```

Comment:
Good catch! <3
Please make this fixup to the commit that introduced this line though